### ISSUE TITLE
Add Expectations Page

### ISSUE THREAD

**Issue by deltzabar** [CSSUoB/cssuob.github.io#398]
Ok so here's my vision
A page that has basics about what to expect about different categories of event, to make it easier for people to plan, especially aimed at improving accessibility. That might be information about for example, how board games is normally a free-form event where you can join a group and play games, pizza is normally served halfway through & will be organised through queueing, etc.
Basically, should have different categories of event, and information about them. Aiming to be ready to freshers next year :)

**Comment by MattyTheHacker** [comment]
related: #374
